ru/lessons/basics/control-structures.md

```diff
@@ -173,3 +173,21 @@ with
   {:ok, jwt, full_claims} <- Guardian.encode_and_sign(user, :token),
   do: important_stuff(jwt, full_claims)
 ```
+
+Начиная с версии Elixir 1.3, конструкция `with` также начала поддерживать `else`:
+
+```elixir
+import Integer
+
+m = %{a: 1, c: 3}
+
+a = with {:ok, res} <- Map.fetch(m, :a),
+  true <- Integer.is_even(res) do
+    IO.puts "Divided by 2 it is #{div(res, 2)}"
+else 
+  :error -> IO.puts "We don't have this item in map"
+  _ -> IO.puts "It's not odd"
+end
+```
+
+Это помогает структурировать код обработки ошибок с помощью сопоставления с образцом в общем блоке-обработчике. Значение, которое туда передаётся - первое же выражение, которое не сопоставилось в основном теле `with`.
```

Comment:
Thanks, @brain-geek!
` -` should be replaced with `&mdash;`. The rest looks fine.